### PR TITLE
fix: add org avatar to org booking pages [CAL-2333]

### LINKED ISSUE #10689: [CAL-2333] Orgs: Public booking page for an individual doesn't show team or org context
Public booking page looks like a regular individual booking page:

Should be. 

[View in Figma](https://www.figma.com/file/mWENDNaJPdRfZMtA75MghZ/Organizations?type=design&node-id=3468%3A49273&mode=dev)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2333](https://linear.app/calcom/issue/CAL-2333/orgs-public-booking-page-for-an-individual-doesnt-show-team-or-org)</sub>

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Added the org avatar in booking pages under the org. Also fixed link of user avatars'.

<kbd>image</kbd>

Fixes #10689

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to an event type belonging to a user or team within an org and see the org avatar in the list of members as well as checking the links of those avatars to be correct now.

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Heads up this isn't working with org avatars that are not the default one
### On Shell
CleanShot 2023-08-19 at 21 57 13@2x

### On booking page
CleanShot 2023-08-19 at 21 58 06@2x

File: packages/features/bookings/components/event-meta/Members.tsx [hunk 1]
```diff
@@ -37,9 +39,21 @@ export const EventMembers = ({ schedulingType, users, profile }: EventMembersPro
     title: `${user.name}`,
     image: "image" in user ? `${user.image}` : `/${user.username}/avatar.png`,
     alt: user.name || undefined,
-    href: user.username ? `${CAL_URL}/${user.username}` : undefined,
+    href: user.username
+      ? `${entity.orgSlug ? getOrgFullDomain(entity.orgSlug) : CAL_URL}/${user.username}`
+      : undefined,
```
Comment (hariombalhara) [inline packages/features/bookings/components/event-meta/Members.tsx:44]:
```suggestion
    href: `/${user.username}`,
```

Comment (hariombalhara) [reply packages/features/bookings/components/event-meta/Members.tsx:44]: There is `userBookerUrl` hook to do that but I think we can just use relative URL here as this component is meant only for Booker pages

**Review by hariombalhara — DISMISSED** [review]
Left a comment, otherwise looks good to me.

File: packages/features/bookings/components/event-meta/Members.tsx (hunk 1 above)
Comment (leog) [reply packages/features/bookings/components/event-meta/Members.tsx]: Good point, reverted, thanks @hariombalhara

**Comment by leog** [comment]
> Heads up this isn't working with org avatars that are not the default one

@joeauyeung nice catch! using web app URL to reference the org avatar, now it's working as expected, thanks!

**Review by joeauyeung — DISMISSED** [review]
LGTM. Tested with user and team events.

**Review by joeauyeung — APPROVED** [review]

**Pull request merged**